src/interpreter/value.cc

```diff
@@ -97,8 +98,8 @@ namespace verona::interpreter
   {
     if (tag != UNINIT)
     {
-      std::cerr << "Dropped an initialized Value" << std::endl;
-      abort();
+      // TODO: Should this be an internal error?
```

Comment:
I think there is no way for bad bytecode to trigger this, unless it’s a VM bug, so internal error seems fine to me.

There’s actually an explicit check in the return opcode to make sure all registers in the frame have already been cleared, in which case it calls `fatal`.